addons/ldtk-importer/src/util/tile-util.gd

```diff
@@ -1,5 +1,64 @@
 @tool
 
+# Flip a vector based on a bitset
+static func _flip_vector_array_with_bitset(
+	vecs: PackedVector2Array,
+	bitset: int
+) -> PackedVector2Array:
+	var new_vecs = PackedVector2Array(vecs)
+	for point_idx in range(vecs.size()):
+		var new_vec = Vector2(vecs[point_idx])
+		if bitset & 1:
+			new_vec.x = -new_vec.x 
+		if bitset & 2:
+			new_vec.y = -new_vec.y
+		new_vecs[point_idx] = new_vec
+	return new_vecs
+
+# Copy over and rotate extra tiledata
+static func copy_and_modify_tile_data(
+	tile_data: TileData,
+	orig_tile_data: TileData,
+	physics_layers_cnt: int,
+	navigation_layers_cnt: int,
+	occluder_layers_cnt: int,
+	bitset: int,
+):
+	# Copy over physics
+	for pli in range(physics_layers_cnt):
+		var polygon_cnt = orig_tile_data.get_collision_polygons_count(pli)
+		for pi in range(polygon_cnt):
+			tile_data.add_collision_polygon(pli)
+			var points: PackedVector2Array = _flip_vector_array_with_bitset(orig_tile_data.get_collision_polygon_points(pli, pi), bitset)
+			tile_data.set_collision_polygon_points(pli, pi, points)
+		tile_data.set_constant_angular_velocity(pli, orig_tile_data.get_constant_angular_velocity(pli))
+		tile_data.set_constant_linear_velocity(pli, orig_tile_data.get_constant_linear_velocity(pli))
```

Comment:
Should these also be flipped? 

I never used them in Godot, so I don't know what they do...

Reply:
Reading through the documentation, it seems to make sense for linear velocity. The angular velocity does not have a direction, so its left untouched.